src/components/Login/Login.js

```diff
@@ -80,15 +80,36 @@ class Login extends React.Component {
         <article className="main-panel">
           <header>
             <h1 className="logo-doppler-new">Doppler</h1>
-            <small className="content-signin">
-              {this._('login.you_want_create_account')}{' '}
-              <Link to="/signup" className="link-green uppercase">
-                {this._('login.signup')}
-              </Link>
-            </small>
+            <div className="language-selector">
```

Comment:
What do you think about also apply it in Signup page and also refactoring it as a component?

Reply:
you talk always small tasks and small changes 🤣  but i can

Reply:
You can do it in another PR, no problem.

Just take into account that the task of "select language in non-authenticated pages" will be only completed if all non-authenticated pages have this option.